### PR TITLE
Updated the README.md file to describe the repo & added content to portfolio page

### PR DESCRIPTION
**Pull request by dihim**
Changed the contents within the README.md file to contain the
required format of "This repo contains [name] portfolio and STEP projects."

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,8 +8,12 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>David Ihim's Portfolio</h1>
+        <p>
+            Hi! My name is David Ihim, and I was born and raised in Tulsa, Oklahoma. However, my parents are immigrants from Nigeria.
+            I am currently a rising junior at Stanford. In my free time, I enjoy playing basketball, listening to music, and working out.
+        </P>
```
Comment (ElanHR) [inline portfolio/src/main/webapp/index.html]: nit: < / p >

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -8,8 +8,12 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>David Ihim's Portfolio</h1>
+        <p>
```
Comment (ElanHR) [inline portfolio/src/main/webapp/index.html:13]: nit: this < p > span should be aligned with the previous < h 1 > to indicate that they are children at the same depth of the DOM tree.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -27,7 +24,12 @@ <h2>Projects</h2>
             <li> Spotfire Visual representation of oil sites based on the location of ONEOK's contracted suppliers in the Oklahoma and Kansas region </li>
             <li> Security application to monitor ONEOK's wifi access points </li>
         </ul>
-        
+        <h2>Contact me!</h2>
+         <a target="_blank" href="https://www.linkedin.com/in/david-ihim-404750143"> LinkedIn </a>
```
Comment (ElanHR) [inline portfolio/src/main/webapp/index.html]: nit: please alight with other < a > spans

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -11,9 +11,58 @@
...
+        <h3> Bucket List</h3>
+        <ul>
+            <li> Travel to Nigeria</li>
+            <li> Learn how to play the Piano</li>
+            <li> Travel to Europe</li>
+            <li> Go Scuba Diving</li>
+        </ul>
+        
+        <h2>Image Gallery</h2>
+        <p> These photos encompass some of my favorite memories.</P>
```
Comment (ElanHR) [inline portfolio/src/main/webapp/index.html]: nit: < / p >

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -11,9 +11,58 @@
...
+            So, my top three basetball players to watch are
+        </p>
+            <ul>
+                <li> James Harden - Houston Rockets </li>
+                <li> Lebron James - Los Angeles Lakers </li>
+                <li> Stephen Curry - Golden State Warriors </li>
+            </ul>
+
+        <h3> Top Music Albums</h3>
+        <p> I love listening to music. These are my personal favorite albums based on of nostalgia, music production, and meaning. </P>
```
Comment (ElanHR) [inline portfolio/src/main/webapp/index.html]: nit: < / p >

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -69,9 +69,45 @@ <h2>Image Gallery</h2>
...
+                <li> Routinely sends list of detectable wifi access point names to database </li>
+            </ul>
+            <h4> Technical Stack</h4>
+            <ul>
+                <li> Python</li>
+                <li> Raspberry Pi</li>
+            </ul>
         </ul>
         <h2>Contact me!</h2>
          <a target="_blank" href="https://www.linkedin.com/in/david-ihim-404750143"> LinkedIn </a>
```
Comment (ElanHR) [inline portfolio/src/main/webapp/index.html]: nit: please align with other children of the same depth

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (dihim) [reply portfolio/src/main/webapp/index.html]: Fixed it!

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (dihim) [reply portfolio/src/main/webapp/index.html:13]: Got it, thanks. Looks like I aligned it in a later commit

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (dihim) [reply portfolio/src/main/webapp/index.html]: Fixed

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (dihim) [reply portfolio/src/main/webapp/index.html]: Fixed

File: portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (dihim) [reply portfolio/src/main/webapp/index.html]: Fixed

File: portfolio/src/main/webapp/index.html (hunk 6 above)
Comment (dihim) [reply portfolio/src/main/webapp/index.html]: A bit confused on this one. Is the `<ul>` tag not inline with the `<h4>` tag above?

Comment (ElanHR) [reply portfolio/src/main/webapp/index.html]: Oh I just meant the `<a>` has some extra spacing compared to the `<h2>`

Comment (dihim) [reply portfolio/src/main/webapp/index.html]: Oh ok gotcha, fixed

**Review by ElanHR — APPROVED** [review]

**Pull request merged**